commons/connector/fitbit/fitbit_sleep_classic.avsc

```diff
@@ -9,6 +9,7 @@
     { "name": "duration", "type": "int", "doc": "Duration at this sleep characteristic in seconds." },
     { "name": "level", "type":
       { "name": "FitbitSleepClassicLevel", "type": "enum", "symbols": ["AWAKE", "RESTLESS", "ASLEEP", "UNKNOWN"], "doc": "Level of sleep as computed by Fitbit."},
-      "doc": "Level of sleep as computed by Fitbit.", "default": "UNKNOWN" }
+      "doc": "Level of sleep as computed by Fitbit.", "default": "UNKNOWN" },
+    { "name": "efficiency", "type": ["null", "int"], "doc": "Calculated sleep efficiency score (in percentage). It is a percentage of the amount of time the user was asleep (and not restless) divided by the time they spent in the bed after initially falling asleep.", "default": null }
```

Comment:
It's nice we're going to collect this now. 👍

Reply:
yes its present in both.